app/code/Bulbulatory/Recomendations/Helper/Data.php

```diff
@@ -0,0 +1,18 @@
+<?php
```

Comment:
Dobrze jest nazywać helpery zgodnie z ich przeznaczeniem - tutaj helper jest odpowiedzialny za pobieranie konfiguracji, więc plik mógłby się nazywać `Config.php` i klasa `Config`. Jeśli stwierdzisz, że potrzebny jest helper robiący jakieś inne rzeczy, to stworzysz nowy, odpowiednio nazwany plik, i tam umieścisz pożądane funkcje. Dobrze jest to rozdzielać i odpowiednio nazywać, ponieważ nie zawsze wszystko jest potrzebne i takie rozbicie pozwala na wstrzykiwanie do innych klas tylko tego, co aktualnie potrzebne, a odpowiednie nazewnictwo ułatwia późniejsze utrzymanie kodu.